styles2.css

```diff
@@ -657,3 +542,267 @@ footer > div.contentline {
+    width: 436px;
+    padding-top: 20px;
+  }
+
+  .content > p,
+  article.text > p {
+    height: auto;
+  }
+
+  #main section.icon {
+    margin: 24px auto 74px auto;
+  }
+
+  .contact-form button {
+    margin-left: 0;
+  }
+
+  section.headline > button {
+    margin-bottom: 184px;
+    margin-top: 0;
+  }
+
+  /* Recent work section */
+
+  section.recentWork {
+    width: auto;
+    height: auto;
+    padding: 84px 8px;
+    display: grid;
+    grid-template-columns: 369px 369px 369px;
+    align-items: center;
+    margin-left: auto;
+    margin-right: auto;
+    justify-content: space-between;
+    row-gap: 24px;
+    column-gap: 32px;
+    margin-bottom: 28px;
+  }
+
+  div.title {
+    grid-column: 1/span 3;
+    margin-bottom: 20px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: -4px;
+  }
+
+  section.aboutMe > article > div,
+  section.aboutMe > article > p {
+    max-width: 920px;
+    width: 64%;
+  }
+
+  section.recentWork > div.rHeader.title > div {
+    display: none;
+  }
+
+  h1.title > br {
+    display: none;
+  }
+
+  section.recentWork > article {
+    max-width: 369px;
+  }
+
+  ul.language {
+    width: auto;
+  }
+
+  .vide {
+    height: 252px;
+  }
+
+  section.aboutMe {
+    height: auto;
+    padding: 80px 0;
+    column-gap: 48px;
+  }
+
+  section.aboutMe > article.text {
+    width: inherit;
+    background-image: url("illustration\ about\ me\ desktop\ \(2\).svg"), url(illustration-desktopText.svg);
+    background-position: top right, bottom 16px left;
+    background-size: contain, auto;
+    flex-direction: column;
+    align-items: center;
+    align-content: space-around;
+    gap: 24px;
+  }
+
+  section.aboutMe > article > p {
+    height: auto;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  section.aboutMe > article.text > button {
+    margin-top: 0;
+  }
+
+  ul.maintain {
+    margin-top: 0;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 8px;
+    gap: 24px;
+    width: 1171px;
+    height: auto;
+  }
+
+  div.know {
+    width: 100%;
+    height: 352px;
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    gap: 0;
+    padding: 0 16px;
+    padding-bottom: 24px;
+  }
+
+  div.know > div.contain {
+    margin-top: 0;
+    margin-bottom: 64px;
+    padding-bottom: 0;
+  }
+
+  div.know > div.contain > div.empty {
+    margin-top: 40px;
+    margin-bottom: 0;
+  }
+
+  div.know > div.describe {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+    gap: 24px;
+    margin-bottom: 0;
+  }
+
+  /* Hide footer mobile */
+
+  .mobile {
+    display: none;
+  }
+
+  .desktop {
+    display: block;
+  }
+
+  .contact-form {
+    display: grid;
+    grid-template-columns: 436px 1fr;
+    align-items: flex-start;
+    padding: 80px 0;
+    gap: 24px;
+    width: 1171px;
+    margin: 0 auto 145px auto;
+    background-image: none;
+  }
```

Comment:
- [ ] The contact form should not create a `horizontal scroll` at any `breakpoint` :exclamation: . You should ensure that the width of the form section is responsive to stick to the Figma design. This makes the website look professional :+1: 
- Your design :sunglasses: 
![Screenshot from 2023-01-20 06-47-19](https://user-images.githubusercontent.com/99394374/213613494-80e37de9-2aab-4f1b-ab60-a8f100423a6b.png)